app/Post_Types/Diviner_Field/Preset_Fields_List_Table.php

```diff
@@ -12,25 +12,30 @@
  */
 class Preset_Fields_List_Table extends \WP_List_Table
 {
-	/**
-	 * Prepare the items for the table to process
-	 *
-	 * @return Void
-	 */
-	public function prepare_items()
-	{
-		$columns = $this->get_columns();
-		$hidden = $this->get_hidden_columns();
-		$sortable = $this->get_sortable_columns();
-		$fields = $this->get_fields();
-		$this->_column_headers = array($columns, $hidden, $sortable);
-		$this->items = $fields;
-	}
+    /**
+     * Prepare the items for the table to process
+     *
+     * @return Void
+     */
+    public function prepare_items()
+    {
+        $columns = $this->get_columns();
+        $hidden = $this->get_hidden_columns();
+        $sortable = $this->get_sortable_columns();
+        $fields = $this->get_fields();
+        $this->_column_headers = array($columns, $hidden, $sortable);
+        $this->items = $fields;
+    }
+
+    public function is_empty() {
+        return (bool) ( isset( $fields ) && count( $fields ) > 0 );
+    }
 
 	public function get_fields()
 	{
 		$args = [
-			'post_type' => \Diviner\Post_Types\Diviner_Field\Diviner_Field::NAME
+			'post_type' => \Diviner\Post_Types\Diviner_Field\Diviner_Field::NAME,
```

Comment:
Use `use` and shorten all these fully qualified class calls

Reply:
done